docs/content/test/WorkManager.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Work Manager Testing
+type: docs
+---
+
+# Work Manager Testing
+
+This documents describes how to test WorkManager's Workers.
+
+## WorkManager is disabled for tests
+
+Because using real WorkManager while testing involves a lot of problems as it starts to initialize 
+just before `AvitoTestApp` (using ContentProvider mechanism). Sometimes it leads to a classic 
+race condition: some Worker tries to get its dependencies to build its own Dagger graph but 
+Application isn't ready so test runner ends up with a crash.
+
+## How to test WorkManager's Workers?
```

Comment:
Let's move this explanation to the start. It's a real answer.

Reply:
Agree. Move it closer to the beginning of the document.